test/Common/SupportedLanguagesTestAttribute.cs

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+using System;
+using System.Collections.Generic;
+using Xunit.Sdk;
+using System.Reflection;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Sql.Tests.Common
+{
+    /// <summary>
+    /// SqlInlineData attribute class for adding language parameter data to tests. This allows
+    /// tests to be run against all the supported languages list that are specified in the SupportedLanguages
+    /// list of this class (ex: CSharp, JavaScript)
+    /// </summary>
+    public class SqlInlineDataAttribute : DataAttribute
+    {
+        private readonly Dictionary<string, List<List<object>>> testData = new Dictionary<string, List<List<object>>>();
+        private readonly string _propertyName = "data";
+
+        /// <summary>
+        /// Load supported languages as test parameters data for the test.
+        /// </summary>
+        public SqlInlineDataAttribute()
+        {
+            var langData = new List<List<object>>();
+            foreach (string lang in Enum.GetNames(typeof(SupportedLanguages)))
+            {
+                var listOfValues = new List<object>(1) { lang };
+                langData.Add(listOfValues);
+            }
+            this.testData.Add(this._propertyName, langData);
+
+        }
+        /// <summary>
+        /// Load the test parameters data for all supported languages for inline use of the test.
```

Comment:
`Adds a language parameter to the test data which will contain the language that the test is currently running against from the list of supported languages`